src/reconstruction/modifications/generic.jl

```diff
@@ -2,9 +2,9 @@
     with_set_bound(args...; kwargs...) 
 
 Specifies a model variant that has a new bound set. Forwards arguments to
-[`set_bound`](@ref). Intended for usage with [`screen`](@ref).
+[`change_bound`](@ref). Intended for usage with [`screen`](@ref).
 """
-with_set_bound(args...; kwargs...) = m -> set_bound(m, args...; kwargs...)
+with_set_bound(args...; kwargs...) = m -> change_bound(m, args...; kwargs...) # TODO: change to with_change_bound? Mirek, what do you think?
```

Comment:
yeah the set/change naming should be unified.

I'd actually vote for `set`. A good naming that we use elsewhere is:
- `set` disregards the previous value and just sets the new one
- `modify` allows you tu supply a function that changes the previous value (ie, `set` is a special case of `modify` that disregards it)
- `change` is semantically somewhere in the uncanny gap where you don't know if the original value is going to be used or not

Reply:
Also, I'm not saying that with_set_bound is a good name. It types well in this order (you start with `with_`, then the operation you want to do is `set` and you want to do that with bounds) but you see, it sounds a bit mechanic. Not sure if we find a better option though.

Reply:
after the voting we decided on `change` :)